src/main/java/com/spotify/confidence/ConfidenceValue.java

```diff
@@ -422,7 +425,7 @@ public ConfidenceValue get(String... path) {
           // todo better error
           throw new IllegalStateException();
         }
-        value = values.getOrDefault(path[i], NULL_VALUE);
+        value = values.getOrDefault(path[i], null);
```

Comment:
What triggered this change?

Reply:
If the key is not found in the value struct, we should return null: NULL_VALUE is actually an allowed value for any existing key in the value struct (i.e. [it indicates to return the default value](https://github.com/spotify/confidence-openfeature-provider-java/blob/b2265a5f290e5dc8a8b9a49481783be4dbd7696c/src/main/java/com/spotify/confidence/ConfidenceTypeMapper.java#L107-L109))